lumen/transforms/sql.py

```diff
@@ -1,32 +1,76 @@
 from __future__ import annotations
 
 import datetime as dt
-import textwrap
+import re
 
-from typing import Any, ClassVar
+from typing import ClassVar
 
-import numpy as np
 import param  # type: ignore
+import sqlglot
```

Comment:
What pulls in sqlglot right now? Do we need to make it an explicit dependency of Lumen?